### ISSUE TITLE
[A11Y] Slides carousel: carousel container needs `aria-live=polite`

### ISSUE THREAD

**Issue by aghassemi** [ampproject/amphtml#5485]
Currently activating the next/previous buttons in the carousel does not announce any changes which is not ideal. As soon as user activates next/previous, they should hear the description of the new item that just became visible. Adding  `aria-live=polite` to the `-amp-slides-container` will achieve this behaviour.

**Comment by aghassemi** [comment]
@cramforce we fixed this during the team meeting as an exercise, but since you were not there, you get the bug :)
